scripts/differ/main.go

```diff
@@ -69,10 +70,10 @@ func run() (changes int, err error) {
 	}
 
 	// Prepare the transformers.
-	transformers := Transformers{
-		NoComments(),
-		NoPackageNames(config.ReducedPackageNames),
-		NoEmptyLines(),
+	transformers := transformer.Transformers{
```

Comment:
sorry but this is not what I meant, I meant maybe this change shouldn't be part of this PR..